.storybook/main.ts

```diff
@@ -16,14 +16,31 @@
  * limitations under the License.
  */
 
-module.exports = {
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+const config: StorybookConfig = {
   stories: ['../examples/**/stories.@(ts|tsx|js|jsx)'],
   addons: [
     '@storybook/addon-a11y',
     '@storybook/addon-actions',
     '@storybook/addon-viewport',
+    '@storybook/addon-docs',
   ],
-  core: {
-    builder: 'webpack5',
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {},
+  },
+  docs: {
+    autodocs: false,
+    defaultName: 'Documentation',
+  },
+  typescript: {
+    reactDocgenTypescriptOptions: {
+      shouldIncludePropTagMap: true,
+    },
+  },
+  features: {
+    storyStoreV7: false, // TODO: Remove this. Temporarily opt out of on-demand story loading as it blocks Percy from running tests
```

Comment:
Do we have a ticket number to be able to refer to in the backlog to fix?

Reply:
no, haven't created a ticket. To be honest we're kinda blocked on this while Storybook sorts out this issue and properly supports Percy